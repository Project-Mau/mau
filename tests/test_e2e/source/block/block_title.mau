. The title
----
This is a block
----
