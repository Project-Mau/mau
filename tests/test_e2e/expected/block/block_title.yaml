data:
  args: []
  content:
  - data:
      args: []
      classes: []
      content:
      - data:
          args: []
          content:
          - data:
              args: []
              kwargs: {}
              subtype: null
              tags: []
              type: text
              value: This is a block
          kwargs: {}
          subtype: null
          tags: []
          title: {}
          type: paragraph
      engine: null
      kwargs: {}
      preprocessor: none
      secondary_content: []
      subtype: null
      tags: []
      title:
        data:
          args: []
          content:
          - data:
              args: []
              kwargs: {}
              subtype: null
              tags: []
              type: text
              value: The title
          kwargs: {}
          subtype: null
          tags: []
          type: sentence
      type: block
  kwargs: {}
  subtype: null
  tags: []
  type: document

